### PR TITLE
misc improvements

### PR DESCRIPTION
**Pull request by Anton-4**

### CODE REVIEW

File: .cargo/config.toml [hunk 1]
```diff
@@ -38,13 +38,21 @@ ROC_PRINT_UNDERIVABLE                  = "0"
...
+ROC_PRINT_IR_AFTER_DROP_SPECIALIZATION = "0"
 ROC_DEBUG_ALIAS_ANALYSIS               = "0"
+ROC_PRINT_RUNTIME_ERROR_GEN            = "0"
 ROC_PRINT_LLVM_FN_VERIFICATION         = "0"
+ROC_WRITE_FINAL_WASM                   = "0"
+ROC_LOG_WASM_INTERP                    = "0"
 ROC_PRINT_LOAD_LOG                     = "0"
+ROC_SKIP_SUBS_CACHE                    = "0"
+ROC_PRINT_BUILD_COMMANDS               = "0"
+ROC_PRINT_BUILD_COMMANDS_WITH_ENV_VARS = "0"
```
Comment (Anton-4) [inline .cargo/config.toml:58]: This was not up to date.

File: crates/compiler/build/src/link.rs [hunk 2]
```diff
@@ -1439,18 +1446,20 @@ fn run_build_command(mut command: Command, file_to_build: &str, flaky_fail_count
 
 /// Stringify a command for printing
 /// e.g. `HOME=~ zig build-exe foo.zig -o foo`
-fn stringify_command(cmd: &Command) -> String {
+fn stringify_command(cmd: &Command, include_env_vars: bool) -> String {
```
Comment (Anton-4) [inline crates/compiler/build/src/link.rs:1449]: The env vars used to be printed by default but this was very verbose.

**Review by rtfeldman — APPROVED** [review]
Nice, looks good!

**Comment by Anton-4** [comment]
Thanks @rtfeldman :heart:

**Pull request merged**